tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1606,7 +1606,13 @@ def _reset_allow_build_at_runtime(node):
     signatures = {self._input_saved_model_signature_key: self._converted_func}
     save.save(trackable, output_saved_model_dir, signatures, options=options)
 
-  def summary(self, line_length=160, detailed=True, print_fn=None):
+  @deprecation.deprecated_arg_values(
+      None,
+      "Consider using instead: `summary(..., verbosity_lvl=[0..2], ...)`",
+      warn_once=True,
+      detailed=True)
+  def summary(self, line_length=160, detailed=False, verbosity_lvl=1,
+              print_fn=None):
```

Comment:
The new argument should be at the end to avoid breaking users who are not using keywords to specify the arguments. 

If someone used this API as follows: 'foo.summary(100, True, lambda x..)' before this change, then this change would break them.